wolfcrypt/test/test.c

```diff
@@ -2080,7 +2080,7 @@ int poly1305_test(void)
     byte     tag[16];
     Poly1305 enc;
 
-    const byte msg[] =
+    static const byte msg[] =
```

Comment:
MDK compiler does not seem to accept pointers to automatic value for initial values, as such 
in line 2177. I assume this is because the address is actually resolved at run time.
We can either make them static values or explicitly set pointers at run time.

line 2177: const byte* msgs[] = {msg, msg2, msg3};

Kojo